### PR TITLE
Readme updated and added necessary keyword labels and deleted unwanted labels

### PR DESCRIPTION
**Pull request by KartikKar19**

### CODE REVIEW

**Review by N3v1 — APPROVED** [review]
@SID262000 You can merge this PR now

cc: @Devansh-Singh-Parmar, @KartikKar19

**Comment by Devansh-Singh-Parmar** [comment]
please make the DeliverEase logo better ref.

cc: @KartikKar19 @N3v1 
@SID262000 Please do not merge it right now before this thing get done

**Comment by Devansh-Singh-Parmar** [comment]
Always add issue number in the description when you create a pull request
cc: @KartikKar19 

This is issue #32

**Comment by N3v1** [comment]
> please make the DeliverEase logo better ref.
> 
> cc: @KartikKar19 @N3v1 @SID262000 Please do not merge it right now before this thing get done

@Devansh-Singh-Parmar 
I think that's because @KartikKar19 changed the width.
The image should be at a width 100. The height is not needed, I think
```HTML
<img align="right" width="100" src="https://gcdnb.pbrd.co/images/nlDr0mgn0Nkp.png"></a>
```

cc: @SID262000, @KartikKar19

File: README.md
Comment (N3v1) [inline README.md]: @KartikKar19 
- width should be 100
- delete height

I think that'll solve the issue with the logo

Comment (N3v1) [reply README.md]: Insert this code instead of the code in line 3:
```HTML
<img align="right" width="100" src="https://gcdnb.pbrd.co/images/nlDr0mgn0Nkp.png"></a>
```

**Comment by SID262000** [comment]
Let me once this PR is ready, will merge it post confirmation.
cc: @N3v1 @KartikKar19 @Devansh-Singh-Parmar

File: README.md
Comment (Devansh-Singh-Parmar) [reply README.md]: right please do it 
cc @KartikKar19

**Comment by Devansh-Singh-Parmar** [comment]
@N3v1 that's totally right
waiting for @KartikKar19

**Pull request closed without merging**
